.github/CODEOWNERS

```diff
@@ -2,3 +2,4 @@
 * @kevinpschaaf
 /packages/labs/context @benjamind
 /packages/labs/eslint-plugin-lit @43081j
+/packages/rollup-plugin-minify-html-literals @bencbradshaw
```

Comment:
cc/ @kevinpschaaf @justinfagnani (because I do not have privilege to add). Could you add bencbradshaw with write access?

Github error: `Unknown owner on line 5: make sure @bencbradshaw exists and has write access to the repository`